frontend/src/lib/introductions/GroupsIntroductionOption.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import { useValues } from 'kea'
+import { LockOutlined } from '@ant-design/icons'
+import Select from 'rc-select'
+import { Link } from 'lib/components/Link'
+import { groupsAccessLogic, GroupsAccessStatus } from 'lib/introductions/groupsAccessLogic'
+import { IconOffline } from 'lib/components/icons'
+
+export function GroupsIntroductionOption({ value }: { value: any }): JSX.Element | null {
+    const { groupsAccessStatus, upgradeLink } = useValues(groupsAccessLogic)
+
+    if (
+        ![GroupsAccessStatus.HasAccess, GroupsAccessStatus.HasGroupTypes, GroupsAccessStatus.NoAccess].includes(
+            groupsAccessStatus
+        )
+    ) {
+        return null
+    }
+
+    return (
+        <Select.Option
+            key="groups"
+            value={value}
+            disabled
+            style={{
+                height: '100%',
+                width: '100%',
+                overflow: 'hidden',
+                textOverflow: 'ellipsis',
+                backgroundColor: 'var(--bg-side)',
+                color: 'var(--text-muted)',
+            }}
+        >
+            <div
+                style={{
+                    display: 'flex',
+                    alignItems: 'center',
+                    justifyContent: 'space-between',
+                }}
+            >
+                <span>
+                    <LockOutlined style={{ marginRight: 4, color: 'var(--warning)' }} />
+                    Unique groups
+                    <Link
+                        to="https://posthog.com/docs/user-guides/group-analytics?utm_medium=in-product&utm_campaign=group-analytics-learn-more"
+                        target="_blank"
+                        data-attr="group-analytics-learn-more"
+                        style={{ marginLeft: 8, fontWeight: 'bold' }}
+                    >
+                        Learn more
+                    </Link>
+                </span>
+
+                {groupsAccessStatus !== GroupsAccessStatus.HasAccess && (
```

Comment:
Normally I would step on toes and adjust the UI myself but I know you prefer to update your PRs yourself. After some more thought (and @clarkus's comment) I think we should get rid of this icon and just keep the learn more. That's realistically the best outcome.

Reply:
Note that there's no way for us to link to the right "upsell" page from within the docs - we won't know if user is self-hosted or not or their site_url.

This means that users can get stuck.

I propose we iterate the design in a follow-up. Mind taking the lead here (I got pulled into dealing with query perf)?

Reply:
- I don't think we need to know that. We take the user to a very simple section that looks something like this.

```markdown
# Feature availability
Group analytics is a premium feature that requires an additional subscription.
- If you're self-hosting PostHog, Group Analytics is available on the Scale & Enterprise plans. [Get a license now](https://license.posthog.com?utm_source=docs&utm_campaign=group-analytics-docs-license)
- If you're on PostHog Cloud, you just need to set up your billing information. Group Analytics is available on all our paid plans. [Set up billing](https://app.posthog.com/organization/billing?utm_source=docs&utm_campaign=group-analytics-docs-license)
````

- Sounds good, let's merge this then and I can work with @clarkus on a follow-up PR for the design.